keyboards/neson_design/810e/keymaps/default/keymap.c

```diff
@@ -0,0 +1,32 @@
+/**
+ * keymap.c
+ *
+  Copyright 2024 astro
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0]=LAYOUT_default(
+        QK_GESC, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL,          KC_BSPC,   KC_INS, KC_NUM, KC_PSLS, KC_PAST, KC_PMNS,
+        KC_TAB,     KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC,      KC_BSLS,   KC_DEL,  KC_P7,   KC_P8,   KC_P9,
+        KC_CAPS,   KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT,              KC_ENT,            KC_P4,   KC_P5,   KC_P6, KC_PPLS,
+        KC_LSFT,      KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH,   KC_RSFT,               KC_UP,   KC_P1,   KC_P2,   KC_P3,
+        KC_LCTL, KC_LGUI, KC_LALT,                 KC_SPC,                    KC_RALT,MO(1),KC_RCTL,  KC_LEFT,   KC_DOWN, KC_RGHT,   KC_P0, KC_PDOT, KC_PENT),
+
+[1]=LAYOUT_default(
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______,
+        _______, RGB_TOG, RGB_MOD,  KC_F13, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______,
+        RESET, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,                   _______, _______, _______, _______, 
```

Comment:
`RESET` has been deprecated for a very long time now. Please be testing your changes.
```suggestion
        QK_BOOT, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,                   _______, _______, _______, _______, 
```